src/App.tsx

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './components/GoodsList';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export default App;
+  const loadUser = (callback:() => Promise<Good[]>) => {
+    callback()
+      .then(user => {
+        setGoods(user);
+      });
+  };
```

Comment:
Do we load user or something different ?

Reply:
Something different 😄